var/spack/repos/builtin/packages/root/package.py

```diff
@@ -428,8 +433,11 @@ def cmake_args(self):
 
         # Options related to ROOT's ability to download and build its own
         # dependencies. Per Spack convention, this should generally be avoided.
+
+        afterimage_enabled = "+x" if "platform=darwin" not in self.spec else True
```

Comment:
I think that should be...

```suggestion
        afterimage_enabled = ("+x" in self.spec) if "platform=darwin" not in self.spec else True
```